lib/Room.php

```diff
@@ -97,6 +99,8 @@ class Room {
 	public const EVENT_AFTER_SESSION_LEAVE_CALL = self::class . '::postSessionLeaveCall';
 	public const EVENT_BEFORE_SIGNALING_PROPERTIES = self::class . '::beforeSignalingProperties';
 
+	public const DESCRIPTION_MAXIMUM_LENGTH = 250;
```

Comment:
we might need some buffer between table column size and the allowed length.
Also 250 is rather short. If you want to put in a link to a calendar, github project board and some more info you are over it already.

I would say the column size should be 4k and the allowed size 2k or something? Or we even go further and just use `text` and limit it to 32k with this similar to chat messages.

Reply:
cc @jancborchardt

Reply:
> we might need some buffer between table column size and the allowed length.

You are the database master, so just let me know what I have to put in the migration :-P

> Also 250 is rather short. 

Initially the length was larger, but @ma12-co said that the front-end will only allow 250 characters due to design reasons.

Reply:
Well let's make the database save, so we can adjust it easily and don't have to run migrations (especially because changing from string to text doesnt work in oracle